src/Orckestra.Composer.MyAccount/Services/CustomerViewService.cs

```diff
@@ -185,8 +185,17 @@ protected virtual UpdateAccountViewModel GetUpdateAccountViewModel(GetUpdateAcco
         protected virtual List<PreferredLanguageViewModel> GetPreferredLanguageViewModel(CultureInfo currentCulture, string customerLanguage)
         {
             var allcultures = CultureService.GetAllSupportedCultures();
+            CultureInfo customerCultureInfo = null;
 
-            return (from culture in allcultures
+            try
+            {
+                customerCultureInfo = CultureInfo.GetCultureInfo(customerLanguage);
+            }
+            catch
+            {
```

Comment:
Could you please detail why are we doing this? Should we handle somehow exception if we expect it?

Reply:
tests send customerLanguage by which we can't get CultureInfo

Reply:
I think we should send a correct customer culture in a unit test if we validate a successful unit test result, not ignoring wrong data in the code with try-catch. Because this exception will throw every single time when unit tests will be running. Still, on my point of view, it's good to keep this try-catch construction with some handling